src/site/content/es/es.11tydata.js

```diff
@@ -23,6 +23,7 @@ module.exports = function () {
   return {
     lang: lang.lang,
     locale: lang.locale,
+    translated: 'none',
```

Comment:
I'm a little confused by this addition because if I go to localhost:8080/vitals and set my language to spanish I see a banner saying that the page hasn't been translated yet, even though it has.

Reply:
done